libs/core/src/components/pds-layout-row/pds-layout-row.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component, Prop, h, Host} from '@stencil/core';
+
+@Component({
+  tag: 'pds-layout-row',
+  styleUrl: 'pds-layout-row.scss',
+})
+export class PdsLayoutRow {
+  /**
+   * Defines the vertical alignment of the row items.
+   * @defaultValue start
+  */
+  @Prop() alignItems?: `start` | `center` | `end` | `baseline` | `stretch`;
+
+  /**
+   * If `true`, the row will have a bottom.
+   */
+  @Prop() bordered? = false;
+
+  /**
+   * Defines the spacing between the row items.
+  */
+  @Prop() colGap?: string;
```

Comment:
Any thoughts on having the gaps as predefined options? That may be too rigid. Mostly thinking out loud and wondering if they should coordinate with our spacing tokens in some way. 

I may be overthinking it though.

Reply:
Great callout! I've updated this prop's value to reference the tshirt sizes from `pds-layout-box`